.github/actions/dbp-charts/verify-compose/docker_compose.sh

```diff
@@ -3,7 +3,6 @@
 COMPOSE_FILE=$(basename $COMPOSE_FILE_PATH)
 COMPOSE_PATH=$(dirname $COMPOSE_FILE_PATH)
 COMPOSE_BIN="docker compose"
-alf_port=8080
 
 dump_all_compose_logs() {
```

Comment:
we have docker-dump-containers-logs action, we can drop this here and then decide if to integrate it here or rely on upstream workflow to call it